docs/document/content/others/faq/_index.cn.md

```diff
@@ -244,66 +294,16 @@ ShardingSphere中很多功能实现类的加载方式是通过[SPI](https://shar
 - 配置项`max.connections.size.per.query`（默认值为1）调高（版本 >= 3.0.0.M3且低于5.0.0）。
 - 配置项`max-connections-size-per-query`（默认值为1）调高（版本 >= 5.0.0）。
 
-## 19. 如何在inline分表策略时，允许执行范围查询操作（BETWEEN AND、\>、\<、\>=、\<=）？
-
-回答：
-
-1. 需要使用4.1.0或更高版本。
-2. 调整以下配置项（需要注意的是，此时所有的范围查询将会使用广播的方式查询每一个分表）：
- - 4.x版本：`allow.range.query.with.inline.sharding`设置为true即可（默认为false）。
- - 5.x版本：在InlineShardingStrategy中将`allow-range-query-with-inline-sharding`设置为true即可（默认为false）。
-
-## 20. 为什么配置了某个数据连接池的spring-boot-starter（比如druid）和shardingsphere-jdbc-spring-boot-starter时，系统启动会报错？
-
-回答：
-
-1. 因为数据连接池的starter（比如druid）可能会先加载并且其创建一个默认数据源，这将会使得ShardingSphere-JDBC创建数据源时发生冲突。
-2. 解决办法为，去掉数据连接池的starter即可，sharing-jdbc自己会创建数据连接池。
-
-## 21. 在使用sharing-proxy的时候，如何动态在ShardingSphere-UI上添加新的logic schema？
-
-回答：
-
-1. 4.1.0之前的版本不支持动态添加或删除logic schema的功能，例如一个proxy启动的时候有2个logic schema，就会一直持有这2个schema，只能感知这两个schema内部的表和rule的变更事件。
-2. 4.1.0版本支持在ShardingSphere-UI或直接在zookeeper上增加新的logic schema，删除logic schema的功能计划在5.0.0版本支持。
-
-## 22. 在使用ShardingSphere-Proxy时，怎么使用合适的工具连接到ShardingSphere-Proxy？
-
-回答：
-
-1. ShardingSphere-Proxy可以看做是一个mysql server，所以首选支持mysql命令连接和操作。
-2. 如果使用其他第三方数据库工具，可能由于不同工具的特定实现导致出现异常。
-3. 目前已测试的第三方数据库工具如下：
-   - Navicat：11.1.13、15.0.20。
-   - DataGrip：2020.1、2021.1（使用 IDEA/DataGrip 时打开 `introspect using JDBC metadata` 选项）。
-   - WorkBench：8.0.25。
-
-## 23. 引入`shardingsphere-transaction-xa-core`后，如何避免spring-boot自动加载默认的JtaTransactionManager？
+## 25. [其他] 引入`shardingsphere-transaction-xa-core`后，如何避免spring-boot自动加载默认的JtaTransactionManager？
 
 回答:
 
 1. 需要在spring-boot的引导类中添加 `@SpringBootApplication(exclude = JtaAutoConfiguration.class)`。
 
```

Comment:
Same as 17, We think this article is related to `JDBC`, because this problem will only occur when `JDBC` is integrated with `SpringBoot`.

Reply:
> Same as 17, We think this article is related to `JDBC`, because this problem will only occur when `JDBC` is integrated with `SpringBoot`.

ok.